syntax = "proto3";

package test;

service TestService {
  rpc HandleRequest (TestRequest) returns (TestReply) {}
}

message TestRequest {
  repeated uint32 test_id = 1;
}

message TestReply {
  string message = 1;
}
